test/distributed/org/apache/cassandra/distributed/test/InternodeEncryptionEnforcementTest.java

```diff
@@ -103,16 +100,12 @@ public void testOutboundConnectionsAreRejectedWhenAuthFails() throws IOException
             executorService.submit(() -> openConnections(cluster));
 
             /*
-             * instance (1) should not connect to instance (2) as authentication fails;
-             * instance (2) should not connect to instance (1) as authentication fails.
+             * Instance (1) should not be able to make outbound connections to instance (2) but Instance (2) should be
+             * accepting outbound connections from Instance (1)
```

Comment:
The comment is confusing to me.. 
If instance 1 can never make outbound connections, how can instance 2 expect connections _from_ instance 1? Do you mean instance 2 can make outbound connections _to_ instance 1?

Reply:
`RejectOutboundAuthenticator` is configured only on instance 1 of the cluster and `AllowAllAuthenticator` is configured on instance 2. Any outbound connections or inbound connections on instance 2 should be successful as `AllowAllAuthenticator` is used, where as on instance 1, Inbound connections will be accepted but OutboundConnections will be rejected.